src/http_server.rs

```diff
@@ -30,9 +68,19 @@ impl<T: Controller> HttpServer<T> {
 
         let addrs: Vec<_> = self.controller.http_as_addrs().unwrap().collect();
 
-        thread::Builder::new().name("HttpServer".to_owned())
+        debug!("Starting server with cert: {:?} and key: {:?}", cert, key);
+
+        let mut config = TlsSniConfig::new();
+        config.add_ssl_cert("foxbox.local".to_owned(), cert, key).unwrap();
+        config.add_ssl_cert("test.knilxof.org".to_owned(), self.controller.get_remote_tls_crt(), self.controller.get_remote_tls_key()).unwrap();
```

Comment:
Why use SNI at all on the Foxbox? we can just listen with two different https servers, right?